test/SAPHanaSR-hookHelper

```diff
@@ -125,26 +155,40 @@ case "$USECASE" in
     # query CIB and write content to a temporary file to limit the
     # cluster calls to a minimum.
     cibtmp=$(mktemp /tmp/SAPHanaSR_SRHhelper.XXXXXX)
-    cibadmin -Q > "$cibtmp"
-    rc=0
-
-    chk_takeover_attr; rc=$?
-    if [ "$rc" == 4 ]; then
-        # sr_takeover attribute not found or not set to 'T'
-	# check, if multi-state ressource is in maintenance
-        maint=$(chk_msres_maintenance)
-        if [ "$maint" == "true" ]; then
-            # permit sr_takeover, because multi-state ressource is
-            # in maintenance
-	    rc=5
+    cibadmin -Q > "$cibtmp"; rc=$?
+    if [ "$rc" != 0 ]; then
+        case "$rc" in
+        102)
+            # ipc connection to cluster (or parts) lost
+            # permit takeover
+            rc=6
+            ;;
+        *)
+            rc=99
```

Comment:
This would also match the rc 105 which would be returned, if the object just does not exist.

Reply:
yes, that's right. It is related to the 'cibadmin' call failing. I thought in this case only '102' is from special interest and should get a special log message in the hook. All other errors are logged as an 'unspecific error'.
If the 'crm_attribute' call fails, I check for both return codes '102' and '105' and return '6' resp. '4' to deliver the discussed log messages.
If we see the need to differ the log messages for our return code '4' in more detail, we can do. No problem.